ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.11__okd.yaml

```diff
@@ -1,75 +0,0 @@
-base_images:
```

Comment:
Its a bit confusing that this is being done for 4.11 when there are 4.10, etc being left.
Can we remove the entire WMCB repo and WMCO 4.11,4.10, etc?

If so please reframe this as `Remove unsupported versions`